Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -934,17 +935,15 @@
     <!-- XTXT: error dialog - detailed text if device has not enough device space  -->
     <string name="errors_not_enough_device_storage">"Sie haben nicht genug Speicherplatz."</string>
     <!-- XTXT: error dialog - detailed text if there is error with Google API  -->
-    <string name="errors_communication_with_api">"Fehler bei Kommunikation mit Google API"</string>
+    <string name="errors_communication_with_api">"Fehler bei der Kommunikation mit dem Google API"</string>
```

Comment:
API is of course an English acronym, so no rule for the declension of it exists in German, but I think it is more common to decline API as a feminine noun. Thus, the sentence should be:
_Fehler bei der Kommunikation mit **der** Google API_

Reply:
You're right, but I think it would be best to use a word that is not technical, that all people can understand. e.g. "Schnittstelle" instead of "API".

Additionally I was wondering why the German translation is changed in a PR that should only add the Turkish translation. But that is of course another topic.

Reply:
@sapkra our translation tool only updates the translation branch and sometimes some minor changes are then also introduced in a PR like this.

Reply:
@schlotter @sapkra Thank you for your feedback. I changed the German to "Schnittstelle".